lib/plugins/helper/fragment_cache.js

```diff
@@ -1,16 +1,18 @@
 'use strict';
 
+const LRU = require('node-lfu-cache');
+
 module.exports = ctx => {
-  let cache = {};
+  const cache = new LRU(50);
```

Comment:
I have run some test by using `Object.keys(cache).length` to know the size of current themes.

| Theme | Current cache size |
|:---:|:---:|
| [hexo-theme-next](https://github.com/theme-next/hexo-theme-next) | 18 |
| [hexo-theme-suka](https://github.com/SukkaW/hexo-theme-suka) | 18 |
| [hexo-theme-cake](https://github.com/jiangtj/hexo-theme-cake) | 17 |

> Those data are from one-language site. Multi-language site could have larger size of cache.

Reply:
It is best to be exposed to the outside for configuration, If some theme has larger cache length, it can adjust it.

Reply:
Most of users have no idea about the cache size of their themes.

Reply:
> Most of users have no idea about the cache size of their themes.

But as a theme developer. If he knew clearly that he needed 50+, how to achieve it?

Reply:
@jiangtj In fact he/she doesn't. It is better to cache as less as possible because larger size of cache means higher memory usage. It is not a wise idea to cache everything in the `fragment_cache`. That's how LFU works, by only caching the most usefuls.